src/core/tsi/test_creds/README

```diff
@@ -97,3 +124,10 @@ src/python/grpcio_tests/tests/interop/credentials/
 src/python/grpcio_tests/tests/unit/credentials/
 src/ruby/spec/testdata/
 test/core/end2end/data/
+
+These are newly added keys/certs that haven't been distributed through multiple
```

Comment:
nit: please rephrase as

The following keys/certs are not distributed through multiple grpc repos yet, since they are only used in grpc core tests.